mesheryctl/internal/cli/root/perf/apply.go

```diff
@@ -212,6 +216,11 @@ mesheryctl perf apply -f <filepath>
 
 				// reset profile name without %20
 				profileName = response.Profiles[index].Name
+				loadGenerator = response.Profiles[index].LoadGenerators[0]
```

Comment:
Are we pulling this definition from SMP proto? // @navendu-pottekkat @DelusionalOptimist

Reply:
These are referenced from a model struct in Meshery which I guess are not pulled from the spec.

@piyushsingariya We do have a new struct which pulls from the SMP definition right? I think we reference this struct for the performance configuration file.

Reply:
there's no need to use SMP proto we have a struct for profiles.